bin/ch/WScriptJsrt.cpp

```diff
@@ -1080,9 +1100,6 @@ bool WScriptJsrt::Initialize()
     IfFalseGo(WScriptJsrt::InstallObjectsOnObject(wscript, "SerializeObject", SerializeObject));
     IfFalseGo(WScriptJsrt::InstallObjectsOnObject(wscript, "Deserialize", Deserialize));
 
-    // ToDo Remove
-    IfFalseGo(WScriptJsrt::InstallObjectsOnObject(wscript, "Edit", EmptyCallback));
```

Comment:
if we're deleting this can we also delete the EmptyCallback below and from the header?